samples/CommunityToolkit.Maui.Sample/Platforms/Tizen/tizen-manifest.xml

```diff
@@ -1,15 +1,18 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest package="maui-application-id-placeholder" version="0.0.0" api-version="7" xmlns="http://tizen.org/ns/packages">
-  <profile name="common" />
-  <ui-application appid="maui-application-id-placeholder" exec="CommunityToolkit.Maui.Sample.dll" multiple="false" nodisplay="false" taskmanage="true" type="dotnet" launch_mode="single">
-    <label>maui-application-title-placeholder</label>
-    <icon>maui-appicon-placeholder</icon>
-    <metadata key="http://tizen.org/metadata/prefer_dotnet_aot" value="true" />
-  </ui-application>
-  <shortcut-list />
-  <privileges>
-    <privilege>http://tizen.org/privilege/internet</privilege>
-  </privileges> 
-  <dependencies />
-  <provides-appdefined-privileges />
+    <profile name="common" />
+    <ui-application appid="maui-application-id-placeholder" exec="CommunityToolkit.Maui.Sample.dll" multiple="false" nodisplay="false" taskmanage="true" type="dotnet" launch_mode="single">
+        <label>maui-application-title-placeholder</label>
+        <icon>maui-appicon-placeholder</icon>
+        <metadata key="http://tizen.org/metadata/prefer_dotnet_aot" value="true" />
+        <splash-screens />
+    </ui-application>
+    <shortcut-list />
+    <privileges>
+        <privilege>http://tizen.org/privilege/internet</privilege>
+        <privilege>http://tizen.org/privilege/mediastorage</privilege>
+        <privilege>http://tizen.org/privilege/externalstorage</privilege>
```

Comment:
This isn't a good practice. Because if the app doesn't use this feature and reference our lib this permission will be added to their app, and can be refused by app stores.